### PR TITLE
Pull request from feature branch in develop

### PR DESCRIPTION
**Pull request by aristides1000**
Through the following pull request, we request that the possibility of joining the feature branch in the development branch be studied, knowing the knowledge acquired in the authentication of users on different platforms through the use of Rails

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  # Use callbacks to share common setup or constraints between actions.
+  def set_event
+    @event = Event.find(params[:id])
+  end
+
+  # Only allow a list of trusted parameters through.
+  def event_params
+    params.require(:event).permit(:name, :date, :location)
+  end
+end
```
Comment (carlos-ssh) [inline app/controllers/events_controller.rb]: - Kindly remove commented lines.
- Remove unused format.json.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @ad9311 & @aristides1000 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- User without sign in can edit events, kindly fix this.
- When the user sign in will be redirected into the root path.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


[OPTIONAL]
If you feel like you really want to go beyond requirements and learn more about associations, work on these ones

- route under /event/:event_id/invites/new
- Invite model belongs_to :event
- Invite model belongs_to :user
- User model has_many :invitations
- Event model has_many :invitations
- Join model (EventAttendance) is required to make sure that attendance is not created without invitation (e.g. before_create callback usage)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ad9311** [comment]
The previous changes include:
- Users who are not logged-in cannot longer edit or destroy events. Only see them (events#show).
- When users log in or sign up they are redirected to root_path (events#index).
- Remove commented lines and unsed json code in events_controller.rb.
- Remove code that wasn't being used.

**Review by voscarmv — APPROVED** [review]
# Project status: Approved 💯 ✨ 😎 👍 
### You may merge this branch and move forward ⛙🛣️ 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻
# Happy Coding! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**